src/wsiRequest.h

```diff
@@ -0,0 +1,58 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef SRC_WSIREQUEST_H_
+#define SRC_WSIREQUEST_H_
+
+#include <string>
+#include "src/enums.h"
+
+// Structure for wsi2dcm settings
+// inputFile - wsi input
+// outputFileMask - path to save generated files
+// compression - jpeg, jpeg2000, raw
+// startOnLevel, stopOnLevel - limitation which levels to generate
+// retileLevels - number of levels, levels == 0 means number of
+//          levels will be readed from wsi file
+// imageName, studyId, seriesId - required DICOM metadata
+// jsonFile - json file with additional DICOM metadata
+// downsample - for each level, downsample is size
+//              factor for each level
+// tiled - frame organization type
+//           true:TILED_FULL
+//           false:TILED_SPARSE
+// batchLimit - maximum frames in one file
+struct WsiRequest {
```

Comment:
It's fine to just define this struct in wsi2dcm.h. That way someone looking at functions in wsi2dcm doesn't have to jump to another file to understand the WsiRequest struct.

Reply:
done, but not sure how it would help since some who'll use this will have to jump to wsiToDcm.h, it's same as jump to wsiRequest.h